pkgs/applications/networking/cluster/cmctl/default.nix

```
@@ -1,14 +1,19 @@
 { lib, buildGoModule, fetchFromGitHub, installShellFiles }:
 
-buildGoModule rec {
-  pname = "cmctl";
+let
   version = "1.9.1";
+  rev = "4486c01f726f17d2790a8a563ae6bc6e98465505";
+
+in buildGoModule rec {
+  pname = "cmctl";
+  inherit version;
+  inherit rev;
```

```suggestion
buildGoModule rec {
  pname = "cmctl";
  version = "1.9.1";
```

---

My preference:

```suggestion
buildGoModule rec {
  pname = "cmctl";
  version = "1.9.1";

  src = fetchFromGitHub {
    owner = "cert-manager";
    repo = "cert-manager";
    rev = "4486c01f726f17d2790a8a563ae6bc6e98465505";
    sha256 = "1rzm6dn88nc2c8kayg1y9r7gkmbx42s0ph93ji7z56gqqpbqjmk7";
  };
```

Update script will still work this way.